nncf/dynamic_graph/wrappers.py

```diff
@@ -75,16 +76,18 @@ def wrapped(*args, **kwargs):
             tensor_metas = make_tensor_metas(processed_input)
             node = ctx.find_operator_node(tensor_metas, ia_op_exec_context)
 
-            if node is None:
+            args = tuple(processed_input.op_args)
+            kwargs = processed_input.op_kwargs
+            result = operator(*args, **kwargs)
+
+            if isinstance(result, type(NotImplemented)):
+                nncf_logger.debug("Operation {} returned NotImplemented".format(op_name))
+            elif node is None:
                 node = ctx.maybe_add_node(processed_input, tensor_metas, ia_op_exec_context, module_attrs)
 
             if is_debug():
                 ctx.register_node_call(node)
```

Comment:
I suppose that this should also be moved under `if node is not None`

Reply:
done